applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -268,16 +327,24 @@ int handle_at_fota(enum at_cmd_type cmd_type)
 			}
 			err = 0;
 #endif
-		} else if (op == AT_FOTA_ERASE_APP) {
+		} else if (op == SLM_FOTA_APP_READ) {
+			(void)do_fota_app_read(PM_MCUBOOT_PRIMARY_ID);
+			(void)do_fota_app_read(PM_MCUBOOT_SECONDARY_ID);
```

Comment:
The function returns `void`, no need for an extra cast.

Reply:
was returning `int` 😓 fixed